packages/components/file-card/src/file-card.js

```diff
@@ -0,0 +1,357 @@
+						>
+							<g id="Symbols" stroke="none" stroke-width="1">
+								<g
+									id="card/5.-File-card/standard"
+									transform="translate(-262.000000, -22.000000)"
+								>
+									<g id="5.-File-card">
+										<g
+											id="icon/UI-library/clear"
+											transform="translate(258.000000, 18.000000)"
+										>
+											<path
+												d="M4,12 C4,7.581722 7.59071231,4 12,4 C16.418278,4 20,7.59071231 20,12 C20,16.418278 16.4092877,20 12,20 C7.581722,20 4,16.4092877 4,12 Z M12,13.1313708 L15.0343146,16.1656854 L16.1656854,15.0343146 L13.1313708,12 L16.1656854,8.96568542 L15.0343146,7.83431458 L12,10.8686292 L8.96568542,7.83431458 L7.83431458,8.96568542 L10.8686292,12 L7.83431458,15.0343146 L8.96568542,16.1656854 L12,13.1313708 Z"
+												id="Icon"
+											></path>
+										</g>
+									</g>
+								</g>
+							</g>
+						</svg>
+					</span>
+					<span class="download-action action-icon-wrapper icon-wrapper">
+						<svg
+							class="action-icon"
+							width="15px"
+							height="16px"
+							viewBox="0 0 15 16"
+							version="1.1"
+							xmlns="http://www.w3.org/2000/svg"
+							xmlns:xlink="http://www.w3.org/1999/xlink"
+						>
+							<g id="Components" stroke="none" stroke-width="1">
+								<g
+									id="New-Specs"
+									transform="translate(-1002.000000, -1137.000000)"
+								>
+									<g
+										id="Group-5-Copy-6"
+										transform="translate(739.000000, 1085.000000)"
+									>
+										<g
+											id="card/5.-File-card/normal"
+											transform="translate(0.000000, 30.000000)"
+										>
+											<g id="5.-File-card">
+												<g
+													id="icon/UI-library/download"
+													transform="translate(258.000000, 18.000000)"
+												>
+													<path
+														d="M12.1590909,17.3636364 C11.9999992,17.3636364 11.8553726,17.2840917 11.7252066,17.125 L7.34297521,11.571281 C7.25619791,11.4555779 7.21280992,11.3615706 7.21280992,11.2892562 C7.21280992,11.1012387 7.35020524,11.0072314 7.625,11.0072314 L10.7272727,11.0072314 L10.7272727,4 L13.5909091,4 L13.5909091,11.0072314 L16.6931818,11.0072314 C16.9679766,11.0072314 17.1053719,11.1012387 17.1053719,11.2892562 C17.1053719,11.3615706 17.0619839,11.4555779 16.9752066,11.571281 L12.5929752,17.125 C12.4772721,17.2840917 12.3326455,17.3636364 12.1590909,17.3636364 Z M18.8409091,17.8409091 C18.971075,17.8409091 19.0831607,17.8879128 19.1771694,17.9819215 C19.2711782,18.0759302 19.3181818,18.1880159 19.3181818,18.3181818 L19.3181818,19.2727273 C19.3181818,19.4028932 19.2711782,19.5149789 19.1771694,19.6089876 C19.0831607,19.7029963 18.971075,19.75 18.8409091,19.75 L5.47727273,19.75 C5.34710679,19.75 5.23502113,19.7029963 5.1410124,19.6089876 C5.04700366,19.5149789 5,19.4028932 5,19.2727273 L5,18.3181818 C5,18.1880159 5.04700366,18.0759302 5.1410124,17.9819215 C5.23502113,17.8879128 5.34710679,17.8409091 5.47727273,17.8409091 L18.8409091,17.8409091 Z"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 1165 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ALineLength)